src/atomic_file/atomic.rs

```diff
@@ -152,13 +149,22 @@ impl AtomicFile {
     /// version was not the same as `current` and the operation must be retried
     /// with a fresher version of the file. Any other I/O error is forwarded as
     /// well.
+    /// Return the number of old file deleted after swapping
```

Comment:
"Old version number", right?

Reply:
How this number can be used?

Reply:
It's not used. But could be to validate some files are well deleted